odoo/addons/base/module/module.py

```diff
@@ -422,7 +422,8 @@ def module_uninstall(self):
         """
         modules_to_remove = self.mapped('name')
         self.env['ir.model.data']._module_data_uninstall(modules_to_remove)
-        self.write({'state': 'uninstalled', 'latest_version': False})
+        with self.env.norecompute():
+            self.write({'state': 'uninstalled', 'latest_version': False})
```

Comment:
By doing this, you discard all recomputations of fields that depend on `state` or `latest_version`.
There are at least three fields that will not be recomputed: `menus_by_module`, `reports_by_module`, `views_by_module` on model `ir.module.module` itself...

Reply:
🤔 I knew this was going to blow up something...

Do you have any other idea?